profiles/common/modules/custom/tmgmt_poetry/inc/tmgmt_poetry.theme.inc

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * Handles the theming.
+ */
+
+/**
+ * Implements hook_theme().
+ */
+function tmgmt_poetry_theme($existing, $type, $theme, $path) {
+  return [
+    'poetry_callback_answer' => [
+      'variables' => [
+        'demande_id' => NULL,
+        'code' => NULL,
+        'message' => NULL,
+      ],
+      'template' => 'poetry-callback-answer',
+      'path' => TMGMT_POETRY_XML_TEMPLATES_PATH,
```

Comment:
Template files are stored in a directory of the `tmgmt_poetry_mock` module, we should move these files around.